### DISCUSSION TITLE
v4.5.0-0

### DISCUSSION THREAD

**Discussion by electronstudio** [electronstudio/jaylib#40] [Announcements]
Raylib updated to 4.5.

Macos ARM64 build added.

Raspberry Pi build removed.

Untested, especially the Macos ARM64, so please report whether it works.

<hr /><em>This discussion was created from the release <a href='https://github.com/electronstudio/jaylib/releases/tag/v4.5.0-0'>v4.5.0-0</a>.</em>

**Comment by Yeffian** [comment]
MacOS ARM 64 does not work, no windows are created, but it looks like the modules are loading:
```
INFO: Initializing raylib 4.5
INFO: Supported raylib modules:
INFO:     > rcore:..... loaded (mandatory)
INFO:     > rlgl:...... loaded (mandatory)
INFO:     > rshapes:... loaded (optional)
INFO:     > rtextures:. loaded (optional)
INFO:     > rtext:..... loaded (optional)
INFO:     > rmodels:... loaded (optional)
INFO:     > raudio:.... loaded (optional)
```

**Reply by electronstudio** [reply]
Did you use XstartOnFirstThread option? e.g.

    java -XstartOnFirstThread -cp jaylib-4.5.0-0.jar:. Demo

Can you post the full output, or does it hang at that point?

**Reply by Yeffian** [reply]
Hangs after outputting the logs. I'll try to run with that flag you mentioned when I get back to the computer, will keep you posted.

**Reply by Yeffian** [reply]
<img width="826" alt="Screenshot 2024-04-12 at 6 46 52 PM" src="https://github.com/electronstudio/jaylib/assets/76024824/9939ebc1-f6bc-4c76-adf2-dd657d48696d">
I'm using these build configurations on IntelliJ, it still doesn't show anything, hangs after the logs output as last time.

**Reply by electronstudio** [reply]
Intellij could be doing something. Run it from Terminal.

**Reply by Yeffian** [reply]
<img width="769" alt="Screenshot 2024-04-12 at 7 36 22 PM" src="https://github.com/electronstudio/jaylib/assets/76024824/443ea7aa-cfb1-4671-9302-6ca29caa5a02">
I'm not sure whats happening here, java does not find the `Main` class, even though the .class file is right there?

**Reply by electronstudio** [reply]
Perhaps Main is in a package. Try the example program here: https://github.com/electronstudio/jaylib?tab=readme-ov-file#how-to-use-from-command-line

**Reply by Yeffian** [reply]
The flag seems to fix the demo, so thank you for that! I'm just now realising that this flag was also written in the docs, I must've skipped over it, so sorry about that 😅. Thank you for your support and maintaining this project!

**Reply by electronstudio** [reply]
OK, I still have a vague memory that there are issues with running Raylib from IntelliJ on MacOS.  I think I found some work around like hitting alt-tab allowed it open the window; something like that.

**Reply by electronstudio** [reply]
Btw I just released 5.0 compiled on latest macOS ( may be better?)
